packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -191,7 +208,13 @@ export function getAllTaggedElements(
         `[${InspectorModeDataAttributes.ASSET_ID}][${InspectorModeDataAttributes.FIELD_ID}], [${InspectorModeDataAttributes.ENTRY_ID}][${InspectorModeDataAttributes.FIELD_ID}]`,
       );
 
-  const taggedElements: Element[] = [...alreadyTagged];
+  //Spread operator is necessary to convert the NodeList to an array
+  const taggedElements: TaggedElement[] = [...alreadyTagged]
+    .map((element: Element) => ({
+      element,
+      attributes: getInspectorModeAttributes(element, options),
```

Comment:
we could add a property `manuallyTagged: true` here, then we don't need the whole checks on Line 282 :)

Reply:
It already gets added by getInspectorModeAttributes :D I've also renamed this function to make it more clear that this is for manually tagged inspector mode